janus_core/helpers/utils.py

```diff
@@ -191,23 +163,21 @@ def spacegroup(
     )
 
 
-def none_to_dict(dictionaries: list[Optional[dict]]) -> list[dict]:
+def none_to_dict(dictionaries: Sequence[Optional[dict]]) -> Generator[dict]:
```

Comment:
Generator strictly takes 3 args (yield, return, receives), so this should be:
```suggestion
def none_to_dict(dictionaries: Sequence[Optional[dict]]) -> Generator[dict, None, None]:
```